src/oc/math.cpp

```diff
@@ -114,6 +118,11 @@ double errcheck(double d, const char* s) /* check result of library call */
 {
     // errno may not be enabled, rely on FE exceptions in that case. See:
     // https://en.cppreference.com/w/cpp/numeric/math/math_errhandling
+#if defined(NVHPC_CHECK_FE_EXCEPTIONS)
+    std::cout << "errno_enabled = " << errno_enabled << " ; check_fe_except = " << check_fe_except << std::endl;
```

Comment:
`std::cout` -> `std::cerr` ?

But importantly, these messages will be printed on every errcheck call?

Reply:
This is just for test , there is an NVHPC bug ...